speedup/rf_openmp_hmeq.cpp

```diff
@@ -7,7 +7,7 @@
 #include "../src-openmp/random_forest.cpp"
 
 int main(){
-    int ntree = 100;
+    int ntree = 10;
```

Comment:
Remember to change this default when running the script. The default value here should arguably be `ntree = 100`, similar to `sklearn` for consistency.